src/sql/setup.js

```diff
@@ -2,7 +2,7 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-define(["require", "exports"], function (require) {
+define(["require", "vs/base/common/errors", "exports"], function (require, errors) {
```

Comment:
I think imports have to be after require and exports. the loader expects require and exports to be the first two.

Reply:
Ah thanks!